### ISSUE TITLE
Py3.6 System - Scheduled tasks -  snapshot & scrub

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2570]
Post #2567 in testing branch we have a failure within our Web-UI to reflect a successfully executed snapshot task:
Reproducer - Create a snapshot of say the "home" share, limited to 2 copies with a 5 minute schedule. The resulting task is run by crontab but we fail (via our created script and call back, to update our database on the resulting execution and associated info:

```
Jun 05 19:30:01 buildvm CRON[22121]: (root) CMD (/opt/rockstor/.venv/bin/st-snapshot 1 \*-*-*-*-*-*)
Jun 05 19:30:02 buildvm CRON[22119]: (root) CMDEND (/opt/rockstor/.venv/bin/st-snapshot 1 \*-*-*-*-*-*)
...
Jun 05 19:35:01 buildvm CRON[23870]: (root) CMD (/opt/rockstor/.venv/bin/st-snapshot 1 \*-*-*-*-*-*)
Jun 05 19:35:01 buildvm CRON[23869]: (root) CMDEND (/opt/rockstor/.venv/bin/st-snapshot 1 \*-*-*-*-*-*)
```
![no-sched-snap-feedback](https://github.com/rockstor/rockstor-core/assets/2521585/6730434f-49e3-4143-8a48-e3dd67ee3af6)

**Comment by phillxnet** [comment]
We have a potentially corresponding unit test failure as follows:
```
======================================================================
FAIL: test_post_requests_2 (rockstor.storageadmin.tests.test_snapshot.SnapshotTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/rockstor/src/rockstor/storageadmin/tests/test_snapshot.py", line 156, in test_post_requests_2
    self.assertEqual(response.data[0], e_msg)
AssertionError: "Element 'uvisible' must be a boolean, not (<class 'str'>)." != "Element 'uvisible' must be a boolean, not (<type 'unicode'>)."
- Element 'uvisible' must be a boolean, not (<class 'str'>).
?                                              ^^^^^^^^^
+ Element 'uvisible' must be a boolean, not (<type 'unicode'>).
?                                             +++++++++ ^^^

```

**Comment by phillxnet** [comment]
Linking to related code changes having taken place since this issue was created #2579 where import fixes may have affected this issue. Re-testing now.

**Comment by phillxnet** [comment]
We have a running schedule instantiated via Web-UI with consequent triggers from cron evident:
```
Jun 07 18:00:01 buildvm CRON[18867]: (root) CMD (/opt/rockstor/.venv/bin/st-snapshot 2 \*-*-*-*-*-*)
Jun 07 18:00:02 buildvm CRON[18865]: (root) CMDEND (/opt/rockstor/.venv/bin/st-snapshot 2 \*-*-*-*-*-*)
```
But:
```
buildvm:/opt/rockstor # /opt/rockstor/.venv/bin/st-snapshot 2 \*-*-*-*-*-*
Traceback (most recent call last):
  File "/opt/rockstor/.venv/bin/st-snapshot", line 2, in <module>
    from scripts.scheduled_tasks.snapshot import main
  File "/opt/rockstor/src/rockstor/scripts/scheduled_tasks/snapshot.py", line 22, in <module>
    import crontabwindow  # load crontabwindow module
ModuleNotFoundError: No module named 'crontabwindow'
``

**Comment by Hooverdan96** [comment]
again, unrelated to this specifically, but I think there is another instance where the crontab needs to be adjusted:

https://github.com/rockstor/rockstor-core/blob/b04fb035c5ffeead9fa02781667c96032990a0af/src/rockstor/scripts/scheduled_tasks/pool_scrub.py#L23

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks. Yes I'm preparing a pr for both instances now. And have broadened the commit messages accordingly. We are getting there here I think. And these took longer as there was no test coverage !!

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2585
